src/features/Faucet/Faucet.tsx

```diff
@@ -260,8 +258,8 @@ export default function Faucet() {
               </Button>
             </LinkApp>
           )}
-        />
-      )}
+        />)
```

Comment:
??

Reply:
it was just me being nitpicky. i reverted xD